apps/server/src/trpc/routes/mail.ts

```
@@ -6,13 +6,186 @@ import type { DeleteAllSpamResponse } from '../../types';
+        const aliases = await agent.getEmailAliases();
+        aliases.forEach((alias: { email: string; name?: string | null; primary?: boolean }) => {
+          addRecipient(alias.email, alias.name ?? undefined, alias.primary ? 9 : 8);
+        });
+      } catch (e) {
+        console.warn('Failed to fetch email aliases:', e);
+      }
+
+      if (query.trim()) {
+        const lowerQuery = query.toLowerCase().trim();
+        const aliasMatches = Array.from(recipientEmails.values()).filter(
+          (r) => r.email.toLowerCase().includes(lowerQuery) || (r.name && r.name.toLowerCase().includes(lowerQuery)),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)